src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  onTabSelected,
+  selectTabId,
+}) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {
+          tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': tab.id === selectTabId })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => selectTabId !== tab.id
+                  && onTabSelected(tab)}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)